addusers.py

```diff
@@ -421,6 +449,7 @@ def parse_rows(rows, select_user=None):
                     if ks_user:
                         raise ItemExistsError('User', user.name)
                     new_ks_user = openstack.create_user(user, ks_project)
+                    user_dict[new_ks_user.name] = new_ks_user.id
```

Comment:
I added the dictionary here to be able to match the username first to see if the username is a new user